lib/botmetrics.rb

```diff
@@ -36,7 +36,7 @@ def message(team_id:, channel: nil, user: nil, text: nil, attachments: nil)
         "message[channel]"     => channel,
         "message[user]"        => user,
         "message[text]"        => text,
-        "message[attachments]" => attachments
+        "message[attachments]" => attachments.nil? ? nil : attachments.to_json
```

Comment:
maybe we should check if attachments is a string then pass string, otherwise do `to_json`?